vector/src/main/res/values/styles_riot.xml

```diff
@@ -11,6 +11,13 @@
         <item name="android:background">?riotx_background</item>
     </style>
 
+    <style name="ActionModeTheme" parent="Base.Widget.AppCompat.ActionMode">
```

Comment:
You should override `Widget.AppCompat.ActionMode` instead (remove `Base.`).

Have tyou check that it's OK on dark theme?

Reply:
yes

Reply:
thx